bin/broadside

```diff
@@ -89,6 +89,15 @@ command :deploy do |d|
     add_shared_deploy_configs(subcmd)
   end
 
+  d.desc 'Creates an initial task definition and service, then runs a series of bootstrap commands.'
```

Comment:
hmm... the `bootstrap` command actually lives _outside_ the `deploy` command right now: https://github.com/lumoslabs/broadside/blob/5038111f414150138e202d40ccf7f76251507b25/bin/broadside#L45-L48